add symbiosis

allush opened the pull request.

allush commented:
@vrtnd hello, could you tell me how to test my adapter locally?

I had opened a lot PRs to defillama-adapters repository, there is perfect testing tool that allow me to be sure in the PR quality.

Unfortunatelly, here isn't such tool.

vrtnd commented:
@allush hi, fixed your pr and merged https://github.com/DefiLlama/bridges-server/pull/19.
Added readme with info about testsing, thanks for pointing that out

allush commented:
> @allush hi, fixed your pr and merged #19. Added readme with info about testsing, thanks for pointing that out

@vrtnd thanks a lot! could you tell me how to update icon for our bridge? now on web UI it looks like a "smile" :)

also, I will open another one PR with more chains support
